sonaranalyzer-dotnet/its/sources/ManuallyAddedNoncompliantIssues/NetFramework48/NetFramework48.csproj

```diff
@@ -35,6 +35,9 @@
   <ItemGroup>
     <Reference Include="System" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Runtime.Serialization" />
+    <Reference Include="System.Runtime.Serialization.Formatters.Soap" />
+    <Reference Include="System.Web" />
```

Comment:
Since `ObjectStateFormatterTest` was removed from the scope, I think we can remove the `System.Web` reference from the csproj file

Reply:
Indeed in this commit is not needed but I will have to remove it now and add it again in the next PR which handles [LosFormatter](https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.losformatter.-ctor?redirectedfrom=MSDN&view=netframework-4.8#System_Web_UI_LosFormatter__ctor_System_Boolean_System_String_).